virttest/utils_test/qemu.py

```diff
@@ -82,6 +84,31 @@ def get_numa_status(numa_node_info, qemu_pid, debug=True):
     return (qemu_memory, qemu_cpu)
 
 
+def get_link_layer(params, cmd):
+    """
+    Get host link layer
+
+    :param params: Dictionary with the test parameters.
+    :param cmd: Command string
+    """
+    try:
+        utils_path.find_command(cmd)
+    except utils_path.CmdNotFoundError:
+        raise exceptions.TestNAError("Can't find command '%s'" % cmd)
+
+    expected_link_layer = params.get("expected_link_layer",
```

Comment:
InfiniBand is a  network adapter vendor, so it looks rename link_layer to nic_vendor looks more  reasonable